Update markers.js

camdendenson1 opened the pull request:
This includes many hermitcraft season 1 hermit bases. It also includes the game area and the overworked nether hub.

Dinip commented:
I'm a bit confused in what's happening here... You added 11 markers, then added a bit more (to a total of 25), then reverted and now you close the PR? 🤔

camdendenson1 commented:
I had made the first bit then I wanted to add all the others but it said it
had errors so I reverted it and it still says that there was errors so I
deleted it.  I am working on fixing it.

On Sun, Dec 13, 2020, 7:48 AM Dinip <notifications@github.com> wrote:

> I'm a bit confused in what's happening here... You added 11 markers, then
> added a bit more (to a total of 25), then reverted and now you close the
> PR? 🤔
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/Dinip/hermitcraft/pull/39#issuecomment-744010582>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASBELY3DZIQEEF2QVESPZ23SUTA2LANCNFSM4UVYFR7Q>
> .
>
